src/types/callable.js

```diff
@@ -71,12 +71,18 @@ export function parseCallableArgs(info) {
   };
 
   const initOutArgs = () => {
-    return new BigUint64Array(outArgsDetail.map((d) => initArgument(d.type)));
+    const buffer = new ArrayBuffer(8 * outArgsDetail.length);
+    const dataView = new ExtendedDataView(buffer);
+    outArgsDetail.forEach((detail) => {
+      dataView.setBigUint64(initArgument(detail.type));
+    });
```

Comment:
```suggestion
    outArgsDetail.forEach((detail, index) => {
      dataView.setBigUint64(initArgument(detail.type), index * 8);
    });
```

Reply:
I feel like there's a better way to do this.

We might be able to just generate an ArrayBuffer here and pass it to `initArgument` together with an `offset`.